code/modules/power/supermatter/supermatter.dm

```diff
@@ -9,6 +9,7 @@
 #define CO2_HEAT_PENALTY 0.1
 #define NITROGEN_HEAT_PENALTY -1.5
 #define BZ_HEAT_PENALTY 5
+#define H2O_HEAT_PENALTY 13
```

Comment:
As a general comment on the values you've used, some of these are a bit high, why'd you chose them?

Reply:
like i said before, i wanted the h2o to give out a lot of heat when used and almost no power emitted by rads (*wink wink TEG), but i still suck at balancing and this value too needs better testing in a broader range

Reply:
Just keep playing around with it, trust your instincts.